Formula/telnetd.rb

```diff
@@ -0,0 +1,63 @@
+class Telnetd < Formula
+  desc "TELNET server (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/remote_cmds/remote_cmds-54.50.1.tar.gz"
+  sha256 "156ddec946c81af1cbbad5cc6e601135245f7300d134a239cda45ff5efd75930"
+
+  depends_on :xcode => :build
+
+  resource "libtelnet" do
+    url "https://opensource.apple.com/tarballs/libtelnet/libtelnet-13.tar.gz"
+    sha256 "e7d203083c2d9fa363da4cc4b7377d4a18f8a6f569b9bcf58f97255941a2ebd1"
+  end
+
+  def install
+    resource("libtelnet").stage do
+      xcodebuild "SYMROOT=build"
+
+      libtelnet_dst = buildpath/"telnetd.tproj/build/Products"
+      libtelnet_dst.install "build/Release/libtelnet.a"
+      libtelnet_dst.install "build/Release/usr/local/include/libtelnet/"
+    end
+
+    system "make",
+           "-C", "telnetd.tproj",
+           "OBJROOT=build/Intermediates",
+           "SYMROOT=build/Products",
+           "DSTROOT=build/Archive",
+           "CFLAGS=$(CC_Flags) -isystembuild/Products/",
+           "LDFLAGS=$(LD_Flags) -Lbuild/Products/"
+
+    bin.install "telnetd.tproj/build/Products/telnetd"
+    man8.install "telnetd.tproj/telnetd.8"
+  end
+
+  def caveats
```

Comment:
Not sure any of these caveats are needed. We certainly shouldn't be telling people to try & touch `/System` at least post-SIP.

Reply:
@DomT4 Ok, I'll remove them. Though, imho, SIP is overrated and I don't use it myself.

Reply:
> Though, imho, SIP is overrated and I don't use it myself.

It protects standard users pretty well. It has certainly reduced the amount of problems Homebrew sees where people thought it was a brilliant idea to replace `/usr/bin/clang` with a symlink to Homebrew's `clang`, and so on.